Install-DVWA.sh

```diff
@@ -0,0 +1,174 @@
+  ██║██╔██╗ ██║███████╗   ██║   ███████║██║     ██║     █████╗  ██████╔╝
+  ██║██║╚██╗██║╚════██║   ██║   ██╔══██║██║     ██║     ██╔══╝  ██╔══██╗
+  ██║██║ ╚████║███████║   ██║   ██║  ██║███████╗███████╗███████╗██║  ██║
+  ╚═╝╚═╝  ╚═══╝╚══════╝   ╚═╝   ╚═╝  ╚═╝╚══════╝╚══════╝╚══════╝╚═╝  ╚═╝                                                                        
+\033[0m"
+
+welcome_message=$(get_language_message "\033[96mWelcome to the DVWA installer!\033[0m" "\033[96m¡Bienvenido al instalador de DVWA!\033[0m")
+echo -e "$welcome_message"
+
+echo
+# Start of the installer
+
+# Update the repositories
+update_message=$(get_language_message "\e[96mUpdating repositories...\e[0m" "\e[96mActualizando repositorios...\e[0m")
+echo -e "$update_message"
+apt update
+
+# Chek if the dependencies are installed
+dependencies_message=$(get_language_message "\e[96mVerifying and installing necessary dependencies...\e[0m" "\e[96mVerificando e instalando dependencias necesarias...\e[0m")
+echo -e "$dependencies_message"
+
+check_program apache2
+check_program mariadb-server
+check_program mariadb-client
+check_program php
+check_program php-mysqli
+check_program php-gd
+check_program libapache2-mod-php
+check_program git
+
+# Download the repo
+download_message=$(get_language_message "\e[96mDownloading DVWA from GitHub...\e[0m" "\e[96mDescargando DVWA desde GitHub...\e[0m")
+echo -e "$download_message"
+git clone https://github.com/digininja/DVWA.git /var/www/html/DVWA
+sleep 2
+
+# Start MySql
+mysql_start_message=$(get_language_message "\e[96mStarting MySQL...\e[0m" "\e[96mIniciando MySQL...\e[0m")
+echo -e "$mysql_start_message"
+systemctl start mysql.service
+sleep 2
+
+# User Prompt for the MySQL root password
+mysql_root_password=$(get_mysql_root_password)
+
+# Run MySQL commands
+mysql -u root -p$mysql_root_password -e "CREATE DATABASE IF NOT EXISTS dvwa;"
+mysql -u root -p$mysql_root_password -e "CREATE USER 'dvwa'@'localhost' IDENTIFIED BY 'p@ssw0rd';"
+mysql -u root -p$mysql_root_password -e "GRANT ALL PRIVILEGES ON dvwa.* TO 'dvwa'@'localhost';"
+mysql -u root -p$mysql_root_password -e "FLUSH PRIVILEGES;"
+echo
+
+# Success message
+success_message=$(get_language_message "\e[92mMySQL commands executed successfully.\e[0m" "\e[92mComandos MySQL ejecutados correctamente.\e[0m")
+echo -e "$success_message"
+
+sleep 2
+
+# Coping DVWA folder to /var/www/html
+dvwa_config_message=$(get_language_message "\e[96mConfiguring DVWA...\e[0m" "\e[96mConfigurando DVWA...\e[0m")
+echo -e "$dvwa_config_message"
+cp /var/www/html/DVWA/config/config.inc.php.dist /var/www/html/DVWA/config/config.inc.php
+sleep 2
+
+# Assign the right permissions to DVWA
+permissions_config_message=$(get_language_message "\e[96mConfiguring permissions...\e[0m" "\e[96mConfigurando permisos...\e[0m")
+echo -e "$permissions_config_message"
+chown -R www-data:www-data /var/www/html/DVWA
+chmod -R 755 /var/www/html/DVWA
```

Comment:
While doing this you should also set the uploads directory permissions.

Reply:
With chmod -R 755 /var/www/html/DVWA will give all files and folders and subfolders Read, Write and Execution 
![image](https://github.com/digininja/DVWA/assets/47905563/2b73bd2e-658a-484e-8da8-d643949cca22)

Reply:
Ye, I'm used to people installing it as themselves so the ownership is wrong. You set both so it works.